modules/contract/contracts.js

```diff
@@ -0,0 +1,246 @@
+      const keys = this.getKeys(account);
+      if(!keys) {
+        return Helper.outputError(`Couldn't find keys for given account.
+          Please make sure the account exists and added to tezster. Run 'tezster list-accounts to get all accounts`);
+      }
+      const keystore = {
+          publicKey: keys.pk,
+          privateKey: keys.sk,
+          publicKeyHash: keys.pkh,
+          seed: '',
+          storeType: conseiljs.StoreType.Fundraiser
+      };
+    
+      let contractAddress = '';
+      let contractObj = Helper.findKeyObj(config.contracts, contract);
+      if (contractObj) {
+        contractAddress = contractObj.pkh;
+      }
+      
+      if (!contractAddress) {
+        return Helper.outputError(`couldn't find the contract, please make sure contract label or address is correct!`);
+      }
+    
+      try {
+        const result = await conseiljs.TezosNodeWriter.sendContractInvocationOperation(
+                                    tezosNode, keystore, contractAddress, 0, 100000, '', 1000, 100000, undefined, 
+                                    argument, conseiljs.TezosParameterFormat.Michelson);
+        
+        if (result.results) {
+          switch(result.results.contents[0].metadata.operation_result.status) {
+            case 'applied':
+                let opHash = result.operationGroupID.slice(1,result.operationGroupID.length-2);
+                this.addTransaction('contract-call', opHash, keys.pkh, contractObj.label, 0);
+                return Helper.output(`Injected operation with hash ${opHash}`);
+    
+            case 'failed':
+            default:
+                return Helper.outputError(`Contract calling has failed : ${JSON.stringify(result.results.contents[0].metadata.operation_result)}`)
+          }
+        }
+        return Helper.outputError(`Contract calling has failed : ${JSON.stringify(result)}`);
+      }
+      catch(error) {
+        return Helper.outputError(error);
+      }
+  }
+
+  async __getStorage(contract) {
+      let contractAddress = '';
+      let contractObj = Helper.findKeyObj(config.contracts, contract);
+      if (contractObj) {
+        contractAddress = contractObj.pkh;
+      }
+    
+      if (!contractAddress) {
+        return Helper.outputError(`couldn't find the contract, please make sure contract label or address is correct!`);
+      }
+    
+      try {
+        let storage = await eztz.contract.storage(contractAddress);
+        return Helper.output(JSON.stringify(storage));
+      }
+      catch(error) {
+        return Helper.outputError(error);
+      }
+  }
+
+  addContractToConfig(contractLabel, contractAddr) {
+      let contractObj = Helper.findKeyObj(config.contracts, contractLabel);
+      if (contractObj) {
+        return Helper.outputError(`This contract label is already in use. Please use a different one.`);
+      }
+      this.__addContract(contractLabel, contractAddr, '');
+  }
+
+  __addContract(label, opHash, pkh) {
```

Comment:
Why do we need double underscore for function name everywhere? Please remove it from all the places. Only private variables can have single underscore.